presto-main/src/main/java/com/facebook/presto/cost/PlanNodeStatsEstimateMath.java

```diff
@@ -74,10 +74,19 @@ private static SymbolStatsEstimate subtractColumnStats(
         double totalSizeRight = (rightRowCount - nullsCountRight) * rightStats.getAverageRowSize();
         double newNullsFraction = Math.max(nullsCountLeft - nullsCountRight, 0.0) / newRowCount;
         double newNonNullsRowCount = newRowCount * (1.0 - newNullsFraction);
-        StatisticRange range = strategy.range(leftRange, rightRange);
+
```

Comment:
The `com.facebook.presto.cost.PlanNodeStatsEstimateMath#subtractColumnStats` method has no real contract. It does what it does and doesn't promise anything. Add `@Deprecated`